lib/Application.php

```diff
@@ -38,19 +38,11 @@ public function __construct(array $urlParams = []) {
 			$userManager->clearBackends();
 			$userManager->registerBackend($c->query(AlternativeHomeUserBackend::class));
 
-			$userManager->listen('\OC\User', 'postCreateUser', function ($user, $password) use ($c) {
```

Comment:
if this is only for fixing Oracle, I don't see why the code has been moved outside of the "postCreateUser" hook instead of just changing the literal

Reply:
That was from TM patch

What's the intention of this `postCreateUser` here?

Reply:
I think I added this back then to delay updating the accounts table to after the admin user got created, because in some cases it would happen too early. Not sure if this time we got lucky or other core changes would remove the need for this.

If this doesn't break acceptance tests on core then I'm fine keeping it this way.

Reply:
https://drone.owncloud.com/owncloud/core/10273 - acceptance tests are all green